packages/core/lib/asciidoctor/js/opal_ext/quickjs/file.rb

```diff
@@ -0,0 +1,13 @@
+class File
```

Comment:
Could you please open a pull request [upstream](https://github.com/opal/opal/pulls) to contribute this implementation (most likely in: https://github.com/opal/opal/tree/master/stdlib/quickjs)?

Reply:
Sure, here it is https://github.com/opal/opal/pull/2665

Reply:
@ggrossetie It has been merged :tada: 

Let me know what shall I do next (update a package.json deps ?)

Reply:
Neat!

> Let me know what shall I do next (update a package.json deps ?)

Once quickjs support is released, we will need to (carefuly) upgrade Opal and remove `opal_ext/quickjs`.
For now, everything is 👍🏻